packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -87,6 +87,16 @@
     "unmuteGeneric": "Failed to unmute microphone",
     "speakingWhileMuted": "Your microphone is muted",
     "startScreenShareGeneric": "Failed to start screen sharing",
-    "stopScreenShareGeneric": "Failed to stop screen sharing"
+    "stopScreenShareGeneric": "Failed to stop screen sharing",
+    "callNetworkQualityLow": "Network quality is low.",
+    "callNoSpeakerFound": "No speakers or headphones found. Connect an audio device to hear the call.",
+    "callNoMicrophoneFound": "No microphones found. Connect an audio input device.",
+    "callMicrophoneAccessDenied": "Unable to access microphone. Click the lock in the address bar to grant permission to this webpage.",
+    "callMicrophoneMutedBySystem": "You are muted by your system.",
+    "callMacOsMicrophoneAccessDenied": "Unable to access microphone. Grant microphone permission in your macOS privacy settings.",
+    "callLocalVideoFreeze": "Network bandwidth is poor. Your video may appear paused for others on the call.",
+    "callCameraAlreadyInUse": "Unable to access camera. It may already be in use by another application.",
+    "callMacOsCameraAccessDenied": "MacOS is blocking access to your camera. Update your privacy settings to allow this browser to access your camera.",
```

Comment:
For this and others: I wouldn't put the OS in the user facing error - Just that access is blocked

Reply:
Strings from @pereiralex include MacOS specific error strings because some error conditions are specific to that platform.